aspnetcore/fundamentals/logging/loggermessage.md

```diff
@@ -0,0 +1,173 @@
+---
+title: High-performance logging with LoggerMessage in ASP.NET Core
+author: guardrex
+description: Learn how to use LoggerMessage features to create cacheable delegates that require fewer object allocations than logger extension methods for high-performance logging scenarios.
+ms.author: riande
+manager: wpickett
+ms.date: 10/31/2017
+ms.topic: article
+ms.assetid: 1725ffe9-b6be-4fbb-af35-c8b9363e2292
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: fundamentals/logging/loggermessage
+---
+# High-performance logging with LoggerMessage in ASP.NET Core
+
+By [Luke Latham](https://github.com/guardrex)
+
+[LoggerMessage](/dotnet/api/microsoft.extensions.logging.loggermessage) features create cacheable delegates that require fewer object allocations than [logger extension methods](/dotnet/api/Microsoft.Extensions.Logging.LoggerExtensions), such as `LogInformation`, `LogDebug`, and `LogError`. For high-performance logging scenarios, use the `LoggerMessage` pattern.
+
+Logger extension methods require "boxing" (converting) value types, such as `int`, into `object` types and back. The `LoggerMessage` pattern avoids boxing by using static `Action` fields to define logging messages for strongly-typed extension methods. 
```

Comment:
They also have to parse the format string every time you write the message, whereas `LoggerMessage.Define` parses it once when it defines the message.